cfme/tests/test_rest_manual.py

```diff
@@ -94,3 +66,148 @@ def test_notification_url_parallel_requests():
         1700378
     """
     pass
+
+
+@pytest.mark.manual
+@pytest.mark.meta(coverage=[1761836])
+@pytest.mark.tier(3)
+@pytest.mark.parametrize("implementation", ["UI", "Rest"])
+def test_widget_generate_content_via_rest(implementation):
```

Comment:
I'd not add `via_rest` to tests which will be run with several contexts. in addition such contexts will be reflected in test names.

Reply:
My test objective is to test `generate_content` action via REST hence `via_rest` in the name but I also need to check if the action yielded results for both UI and REST, hence the parametrization.